docs/user-guide/federation/federated-ingress.md

```diff
@@ -157,6 +157,17 @@ the network traffic directed to this ingress (i.e. 'Service
 Endpoints' behind the service backing the Ingress), so the Federated Ingress does not yet consider these to
 be healthy shards and will not direct traffic to any of these clusters.
 
+Thirdly, it is worth noting that the federation control system will
```

Comment:
Yeah, that's what I thought too.  I was trying to avoid three "Note that"'s in a row.  But I think I should just turn it all into three numbered notes.